src/encoded/regulome_atlas.py

```diff
@@ -256,7 +273,7 @@ def _regulome_category(self, score_category=None, dataset=None):
             return 'Single_Nucleotides'
         return '???'
 
-    def regulome_evidence(self, datasets):
+    def regulome_evidence(self, datasets, chrom, start, end):
```

Comment:
Should the name of this function stil be regulome_evidence if it's also getting the scores from the bigWig?

Reply:
Scores are part of the features (evidence?) being inputted to the regulome scoring algorithm.